src/Controls/src/Build.Tasks/XamlCTask.cs

```diff
@@ -151,6 +151,8 @@ public class XamlCTask : XamlTask
 		/// </summary>
 		public bool ValidateOnly { get; set; }
 
+		internal bool GenerateFullILInValidateOnlyMode { get; set; }
```

Comment:
Does a `.targets` file need to set this? It could be a private property like `$(_MauiGenerateFullILInValidateOnlyMode)` for now.